test/integration/client.js

```diff
@@ -0,0 +1,17 @@
+const { test } = require('../browser');
+const assert = require('assert');
+
+describe('index page', () => {
+    it('can access index page', test(async (browser, options) => {
+        const page = await browser.newPage();
+        
+        await page.goto(`${options.appUrl}`);
+        await page.waitFor('h2');
+
+        const headerText = await page.evaluate((selection) => {
+            return document.querySelector('h2').innerText;
+        }, 'h2');
```

Comment:
Seems that the `selection` param isn't being used. Would this work if changed to:
```js
const headerText = await page.evaluate(() => document.querySelector('h2').innerText)
```
?

Reply:
Yeah indeed, not needed here. Removed!